e2e-test/tsps_gcp_e2e_test.py

```diff
@@ -131,6 +131,25 @@ def poll_for_imputation_job(tsps_url, job_id, token):
 
     raise Exception(f"tsps pipeline did not complete in 25 minutes")
 
+def query_for_user_quota_consumed(tsps_url, token):
```

Comment:
don't have to change this in your PR, but at some point we should change these (and other similar) to `teaspoons_url`

Reply:
yea lemme make a ticket thats blocked on the service renaming ticket to rename all this stuff - https://broadworkbench.atlassian.net/browse/TSPS-376